soc/arm/nxp_imx/rt/soc_rt10xx.c

```diff
@@ -172,18 +172,14 @@ static ALWAYS_INLINE void clock_init(void)
 #endif
 
 #if DT_NODE_HAS_STATUS(DT_NODELABEL(usb1), okay) && CONFIG_USB_DC_NXP_EHCI
-	CLOCK_EnableUsbhs0PhyPllClock(kCLOCK_Usb480M,
-		DT_PROP_BY_PHANDLE(DT_NODELABEL(usb1), clocks, clock_frequency));
-	CLOCK_EnableUsbhs0Clock(kCLOCK_Usb480M,
-		DT_PROP_BY_PHANDLE(DT_NODELABEL(usb1), clocks, clock_frequency));
+	CLOCK_EnableUsbhs0PhyPllClock(kCLOCK_Usb480M, 0 /* ignored */);
```

Comment:
While the HAL does ignore this parameter (it appears it actually ignores both) I would perfer to provide a valid clock frequency here. The NXP's SDK makes the following API calls:

```
CLOCK_EnableUsbhs0PhyPllClock(kCLOCK_Usbphy480M, 480000000U);
CLOCK_EnableUsbhs0Clock(kCLOCK_Usb480M, 480000000U);
```

Could `usb_hs_phy0` and `usb_hs_phy1` nodes with `clock-frequency = <480000000>;` be used to provide this value in a way that properly uses devicetree? Following the calling convention the SDK uses will make this code easier to maintain if those parameters start being used in a future HAL release

Reply:
Sure, we could provide `clock-frequency = <480000000>;` from DT. I wonder how much value does that bring, as 480M is hardcoded in ton of places in the code, literaly (by number) and in logic (like configuring PLL3/USBPLL to specific value). Maybe we should just provide 480000000 without DT? It is not used anyway, right? But I agree, 480000000 is better than 0 in case NXP SDK uses that value.

Reply:
BTW, rt11xx code passes 24000000 to those functions (from XTAL in DT). And in case of `CLOCK_EnableUsbhs0PhyPllClock()` this value is actually used.

Reply:
I'm fine hardcoding the value without DT as well. Primarily my concern is following the NXP SDK, since that's likely the first codebase a HAL update will be verified against.

Reply:
Hardcoded 480000000.